src/helpers.py

```diff
@@ -1,103 +1,85 @@
+        key (string): the key string in the dictionary to find
+        matches_val (str|int): the str or int to try to find in the key
 
     returns:
         boolean
     """
-    # for use in list , example:
-    # [d for d in list if _matches_dict_id(d, "key", matches_val)]
-    return(dic[f"{key}"] == matches_val)
-    
-def _return_single_dict_match(some_list, match_key, match_val):
-        out = [d for d in some_list if _matches_dict_key_val(d, match_key, match_val)][0]
-        return(out)
+    return(dict_[f"{key}"] == matches_val)
 
 
+def return_single_dict_match(some_list, match_key, match_val):
+    """Return a single match in a dictionary.
 
-def get_rubric_assessment(submission):
-    user = submission["user"]
-    
-    new_dict = {"user_id":  user["_id"],
-           "user_name":  user["name"],
-           "user_sis_id": user["sisId"],
-           "user_score": submission["score"],
-           "submission_attempt": submission["attempt"],
-            "submission_timestamp": submission["submittedAt"],
-            "submission_status": submission["submissionStatus"]}
-    
-    try:
-        rubric_details = submission["rubricAssessmentsConnection"]["nodes"]
-        
-        if rubric_details == []:
-            return(new_dict)
-
-        for i in rubric_details:
-            assessment_ratings = i.get("assessmentRatings")
-            
-            points_list = []
-            descriptive_list = []
-            comments_list = []
-            
-            
-            for j in assessment_ratings:
-                criteria = j["criterion"]["description"]
-                
-                points = {criteria: j.get("points")}
-                descriptive = {criteria: j.get("description")}
-                comment = {criteria: j.get("comments")}
-                
-                points_list.append(points)
-                descriptive_list.append(descriptive)
-                comments_list.append(comment)
-                
-                new_dict.update({criteria: j.get("points")})
-                
-            new_dict.update({"points": points_list,
-                            "descriptions": descriptive_list,
-                            "comments": comments_list})
-            
-            return(new_dict)
-        
-    except Exception as err:
-        print(f"Error: {err}")
+    parameters:
+        some_list (dict)
+        match_key (string): the key string in the dict to find
+        matche_val (str|int): the str or int to try to find in the key
 
+    returns:
+        out (dict): dictionary pair
+    """
+    out = [d for d in some_list if _matches_dict_key_val(d, match_key, match_val)][0]
+    return(out)
 
 
 def __get_assessment_criteria_scores(assessment_rating):
+    # Create a dictionary which includes assessment criteria and points assigned
     criteria = assessment_rating.get("criterion").get("description")
-
     ratings_dictionary = {criteria: assessment_rating.get("points")}
-    
```

Comment:
Can you explain this change to me?

Reply:
Most of the code in this function was commented out and unused, so I deleted the commented out part.

Reply:
didn't even notice the comments